eng/pipelines/runtimelab.yml

```diff
@@ -123,9 +123,9 @@ extends:
                   uploadIntermediateArtifacts: false
                   librariesConfiguration: Release
             ${{ if eq(variables.isOfficialBuild, false) }}:
-              buildArgs: -s clr.aot+libs+nativeaot.packages -c $(_BuildConfig) /p:ArchiveTests=true
+              buildArgs: -s clr.aot+mono.wasmruntime+libs+nativeaot.packages -c $(_BuildConfig) /p:ArchiveTests=true
```

Comment:
@yowl Do you know if `EMSDK` is set when targeting OSX or win-arm? https://dev.azure.com/dnceng-public/public/_build/results?buildId=492090&view=logs&j=752aa75d-2981-5712-2e83-4a413c44ec15&t=831180b0-4fa3-513d-132c-4972ab1af17e